Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Button.css';
+
+function ActionButton(props) {
+    return <button className={props.action.buttonStyle}>{props.action.text}</button>
```

Comment:
You would apply the onClick to each component and have them tied to whichever function you set-up in App.js